api/src/main/scala/org/virtuslab/ideprobe/util/UUIDUtil.scala

```diff
@@ -0,0 +1,19 @@
+package org.virtuslab.ideprobe.util
+
+import java.nio.ByteBuffer
+import java.util.{Base64, UUID}
+
+object UUIDUtil {
```

Comment:
Can we adjust names & packages, so that it can be referred as:
`org.virtuslab.ideprobe.UUIDs.randomUUID()`

Reply:
Done